docs/Permissions.md

```diff
@@ -85,71 +220,34 @@ export default {
 ```
 {% endraw %}
 
-## Getting User Permissions
+### `usePermissions`
```

Comment:
<!-- Notes are always non-blocking and simply highlight something the reader should take note of.  -->
**note (non-blocking):** ‏This reference may could confuse users in the Algolia search between this and the `userPermissions` page

Reply:
You're right, but we may reintegrate the doc for this hook here and delete the standalone chapter, to avoid overwhelming uses with a long list of hooks. In other words, this is a work in progress.